ext/lib/crypto/mbedtls/configs/config-full-mqtt.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
+ * Copyright (c) 2017 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Minimal configuration for TLS 1.2 (RFC 5246) for Zephyr, implementing only
+ * a few of the most popular ciphersuites.
```

Comment:
I wonder if we should be more explicit that this is about adding the support needed to communicate using MQTT. It's obvious from the filename, but not the source.

Reply:
done